app/graphql/mutations/base_mutation.rb

```diff
@@ -0,0 +1,21 @@
+module Mutations
+  class BaseMutation < GraphQL::Schema::RelayClassicMutation
+    private
+
+    def authorize!(resource, action)
+      user = context[:current_user]
+      authorizer = Authorizer.new(user)
+      permission_name = resource.permission_name(action)
+
+      unless authorizer.can?(permission_name, resource)
+        raise GraphQL::ExecutionError.new(
```

Comment:
![unauthorized](https://user-images.githubusercontent.com/2664090/54608216-8a524f80-4a50-11e9-8553-4d8a85f0834c.png)

Works for now, though we should probably think about how to show errors in nice and consistent way in the future. Some errors are displayed in `data.${mutationName}.errors` while others at top level, which gives us different shape of responses. We might consider including all errors in schema as some people [suggest](https://itnext.io/the-definitive-guide-to-handling-graphql-errors-e0c58b52b5e1).